scripts/R56_B13_test.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
-SAVE_PATH="."
+
+# set base paths: customize as needed,
+# or comment out if you already set these up as global environment variables
+export NSDS_METADATA_PATH="${HOME}/Src/NSDSLab-NWB-metadata/"
+export NSDS_DATA_PATH="/clusterfs/NSDS_data/hackathon20201201/TTankBackup/"
+export NSDS_STIMULI_PATH="/clusterfs/NSDS_data/hackathon20201201/Stimuli/"
```

Comment:
You should set these in your local environment (.bashrc or equivalent). Setting them here will break the script for other people.

Reply:
moved to .bashrc